diff --git a/node_modules/react-native-svg/android/src/main/jni/CMakeLists.txt b/node_modules/react-native-svg/android/src/main/jni/CMakeLists.txt
index f21366b..15e6d07 100644
--- a/node_modules/react-native-svg/android/src/main/jni/CMakeLists.txt
+++ b/node_modules/react-native-svg/android/src/main/jni/CMakeLists.txt
@@ -63,6 +63,7 @@ endif()
 target_link_libraries(
   react_codegen_rnsvg
   fbjni
+  c++_shared
 )
 
 target_include_directories(
