diff --git a/node_modules/react-native-sms-user-consent/android/build.gradle b/node_modules/react-native-sms-user-consent/android/build.gradle
index 267e43f..e373bba 100644
--- a/node_modules/react-native-sms-user-consent/android/build.gradle
+++ b/node_modules/react-native-sms-user-consent/android/build.gradle
@@ -1,26 +1,11 @@
-
-buildscript {
-    repositories {
-        jcenter()
-        google()
-    }
-
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.1'
-    }
-}
-
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 34
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 22
-        versionCode 1
-        versionName "1.0"
+        minSdkVersion 24
+        targetSdkVersion 34
     }
     lintOptions {
         abortOnError false
@@ -34,7 +19,6 @@ repositories {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation "com.google.android.gms:play-services-auth:17.0.0"
-    implementation "com.google.android.gms:play-services-auth-api-phone:17.1.0"
+    implementation "com.google.android.gms:play-services-auth:21.0.0"
+    implementation "com.google.android.gms:play-services-auth-api-phone:18.0.2"
 }
-  
\ No newline at end of file
